weather_forecast_retrieval/data/hrrr/file_loader.py

```diff
@@ -50,9 +50,8 @@ def file_type(self):
 
     def get_saved_data(self,
                        start_date, end_date, bbox,
-                       output_dir=None, file_type='grib2',
+                       input_dir=None, file_type='grib2',
```

Comment:
This will also warrant a minor version bump.

Reply:
This is what I was referring to with the second bullet point in the PR description. The sum of changes makes this a major bump as it will break current use of WFR.